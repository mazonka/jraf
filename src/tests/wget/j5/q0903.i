command=jw md 0 /.jraf.sys/users : login admin@server.com http://server/ : profile f82d3bb17612458c : profile 0