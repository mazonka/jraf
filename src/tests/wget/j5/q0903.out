OK /.jraf.sys/users : OK http://server/jraf? : OK a admin@server.com 10000 20170123123456 1 home/34d3ae7c5e2cdebb : OK u * * * * *